django_toolkit/fallbacks/circuit_breaker/__init__.py

```diff
@@ -0,0 +1,116 @@
+# -*- coding: utf-8 -*-
```

Comment:
What if you create a `circuit_breaker.py` file in this folder and use the `__init__.py` file only to import the class from there. Something like:
```python
# __init__.py

from .circuit_breaker import CircuitBreaker
```

For me is common to forget to look for code in `__init__.py`, because usually there's nothing there.